src/lib/dux/sdk/reducers.ts

```diff
@@ -4,30 +4,30 @@ import initialState, { SdkStoreStateType } from './initialState';
 
 export default function reducer(state: SdkStoreStateType, action: SdkActionTypes): SdkStoreStateType {
   return match(action)
-    .with({ type: SDK_ACTIONS.SET_SDK_LOADING }, ({ payload }) => {
+    .with(SDK_ACTIONS.SET_SDK_LOADING, ({ payload }) => {
```

Comment:
So, if we match(action.type), we would still get `({ payload })` ? It didnt work that way sometime before I checked

Reply:
ahh yeah my bad. let me update again